### PR TITLE
chuva-io#17 - Add Portuguese README

### LINKED ISSUE #17: Add Portuguese README
Create `/README` folder with `README_pt.md` with translated content from `README.md`

### PR DESCRIPTION
**Pull request by djackR**

### CODE REVIEW

File: README/README_pt.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# SISP PAYMENTS JS
+
+# Introdução
+Este módulo simplifica o início no processamento de pagamentos Vinti4, Visa e Mastercard usando SISP no Node.js.
+
+**Observação:** Você pode solicitar  as credenciais do SISP, ver documentação e muito mais [aqui](https://www.vinti4.cv/web.aspx).
```
Comment (RonyGomesK78) [inline README/README_pt.md]: plz remove the double spaces between "solicitar" and "as" .

File: README/README_pt.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+### Exemplo
+```js
+const referenceId = 'abc-123';
+const total = 1200;
+const webhookUrl = 'https://samba.chuva.io/webhooks/sisp-payment';
+
+const htmlForm = sisp.generatePaymentRequestForm(referenceId, total, webhookUrl);
+```
+
+## Validar status de processamento de pagamento (método  de conveniência)
```
Comment (RonyGomesK78) [inline README/README_pt.md]: double spaces between "método" and "de"

File: README/README_pt.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+```
+## Gerar formulário de solicitação de pagamento
+```js
+sisp.generatePaymentRequestForm(referenceId, total, webhookUrl);
+```
+Gera e retorna um formulário HTML que pode ser usado para processar pagamentos.
+
+* `referenceId`: Client-generated payment reference ID. Este valor é enviado através de uma requisição  `POST` para o  `webhookUrl` e permite ao cliente correlacionar a solicitação de pagamento e a resposta.
+* `total`: Valor do pagamento (em CVE).
+* `webhookUrl`: A url para onde o SISP deve enviar a resposta de pagamento. Você deve esperar uma requisição `POST` com informações de pagamento no corpo. Veja o SISP [documentation]( https://www.vinti4.cv/documentation.aspx?id=682) para mais informações.
```
Comment (RonyGomesK78) [inline README/README_pt.md]: Plz change "Veja o SISP [documentation] ..." to "Veja a [documentação] (https://www.vinti4.cv/documentation.aspx?id=682) do SISP ..."

**Review by RonyGomesK78 — CHANGES_REQUESTED** [review]

File: README/README_pt.md (hunk 2 above)
Comment (djackR) [reply README/README_pt.md]: ok, will be done

File: README/README_pt.md (hunk 3 above)
Comment (djackR) [reply README/README_pt.md]: ok, will be done

File: README/README_pt.md (hunk 1 above)
Comment (djackR) [reply README/README_pt.md]: ok, will be done

**Review by RonyGomesK78 — DISMISSED** [review]

**Comment by FabyoSK** [comment]
Please squash your commits into one single commit

it should be: 
chuva-io#17 - Add Portuguese README

**Comment by djackR** [comment]
@FabyoSK done

**Review by FabyoSK — APPROVED** [review]

**Review by RonyGomesK78 — APPROVED** [review]

**Review by lcsfort — APPROVED** [review]

**Review by FredSoares — APPROVED** [review]

**Pull request merged**